### DISCUSSION TITLE
Notification list shortcut: not implemented or not working?

### DISCUSSION THREAD

**Discussion by phyhari** [paperwm/PaperWM#969] [General]
"The notification list gives you a way to get back to your notifications when it is convenient for you. It appears when you click on the clock, or press [Super]+V." (https://help.gnome.org/users/gnome-help/stable/shell-notifications.html.en)

This does not seem to work with PaperWM. It would be nice if it would. Does it work for you?

**Comment by Lythenas** [comment]
By default `Super`+`V` is bound to "Center window vertially (non-tiled window) in PaperWM. PaperWM removes all conflicting gnome bindings (temporarily). So the default gnome keybinding would be removed. Note that the keybindings are restored when PaperWM is disabled etc.

If you want to use that keybinding you should be able to just remove it in PaperWM. Then you probably need to logout and log back in for the gnome keybinding to work again.

But I feel like for such a niche keybinding like this the default should not conflict with a default gnome keybinding. So maybe we should change that to not confuse new users.

**Reply by phyhari** [reply]
Thank you very much.  I am a new user indeed. Is there a searchable list of every (default PaperWM) key binding? On the GitHub repo main page I don't see the aforementioned shortcut and in the extension settings /keybindings window the results are only searchable for the description and not the shortcut itself.

**Reply by Lythenas** [reply]
I believe the readme lists most of the default keybindings. Otherwise you can check directly in the dconf schema: https://github.com/paperwm/PaperWM/blob/release/schemas/org.gnome.shell.extensions.paperwm.gschema.xml
